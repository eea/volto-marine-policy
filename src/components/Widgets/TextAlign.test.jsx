import React from 'react';
import renderer from 'react-test-renderer';
import configureStore from 'redux-mock-store';
import { Provider } from 'react-intl-redux';

import TextAlign from './TextAlign';

const mockStore = configureStore();

test('renders a text align component', () => {
  const store = mockStore({
    intl: {
      locale: 'en',
      messages: {},
    },
  });

  const component = renderer.create(
    <Provider store={store}>
      <TextAlign
        id="my-field"
        title="My field"
        value="left"
        fieldSet="default"
        onChange={() => {}}
        onBlur={() => {}}
        onClick={() => {}}
      />
    </Provider>,
  );
  const json = component.toJSON();
  expect(json).toMatchSnapshot();
});
