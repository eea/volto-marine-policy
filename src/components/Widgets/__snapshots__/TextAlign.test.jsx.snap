// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders a text align component 1`] = `
<div
  className="inline field field-wrapper-my-field"
>
  <div
    className="ui grid"
  >
    <div
      className="stretched row"
    >
      <div
        className="four wide column"
      >
        <div
          className="wrapper"
        >
          <label
            htmlFor="field-my-field"
            id="fieldset-default-field-label-my-field"
          >
            My field
          </label>
        </div>
      </div>
      <div
        className="eight wide column"
      >
        <div
          className="align-tools"
        >
          <div
            className="ui buttons"
          >
            <button
              aria-label="left"
              className="ui active basic compact icon button"
              onClick={[Function]}
            >
              <svg
                className="icon"
                dangerouslySetInnerHTML={
                  Object {
                    "__html": undefined,
                  }
                }
                onClick={null}
                style={
                  Object {
                    "fill": "currentColor",
                    "height": "24px",
                    "width": "auto",
                  }
                }
                viewBox=""
                xmlns=""
              />
            </button>
          </div>
          <div
            className="ui buttons"
          >
            <button
              aria-label="right"
              className="ui basic compact icon button"
              onClick={[Function]}
            >
              <svg
                className="icon"
                dangerouslySetInnerHTML={
                  Object {
                    "__html": undefined,
                  }
                }
                onClick={null}
                style={
                  Object {
                    "fill": "currentColor",
                    "height": "24px",
                    "width": "auto",
                  }
                }
                viewBox=""
                xmlns=""
              />
            </button>
          </div>
          <div
            className="ui buttons"
          >
            <button
              aria-label="center"
              className="ui basic compact icon button"
              onClick={[Function]}
            >
              <svg
                className="icon"
                dangerouslySetInnerHTML={
                  Object {
                    "__html": undefined,
                  }
                }
                onClick={null}
                style={
                  Object {
                    "fill": "currentColor",
                    "height": "24px",
                    "width": "auto",
                  }
                }
                viewBox=""
                xmlns=""
              />
            </button>
          </div>
          <div
            className="ui buttons"
          >
            <button
              aria-label="justify"
              className="ui basic compact icon button"
              onClick={[Function]}
            >
              <svg
                className="icon"
                dangerouslySetInnerHTML={
                  Object {
                    "__html": undefined,
                  }
                }
                onClick={null}
                style={
                  Object {
                    "fill": "currentColor",
                    "height": "24px",
                    "width": "auto",
                  }
                }
                viewBox=""
                xmlns=""
              />
            </button>
          </div>
        </div>
      </div>
    </div>
  </div>
</div>
`;
